src/components/shared/TokenInfoPopover/TokenInfoPopoverWrapper.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactNode, useMemo } from 'react';
+import { PopperOptions } from 'react-popper-tooltip';
+
+import Popover from '~shared/Popover';
+import { Token } from '~types';
+
+import TokenInfoPopover from './TokenInfoPopover';
+
+interface TokenContentProps {
+  isTokenNative?: boolean;
+  token?: Token;
```

Comment:
Why is `token` optional here? If `token` is `undefined`, this popover's content is `null`, which means there'd be no reason to render it. Right? This is as much me thinking out loud since I may not be following the code correctly, but if that's correct then lines 36-42 could be removed and this component only rendered if the token prop is defined.

Reply:
The token functionality was not the focus of this PR and I can't test the Token info right now but wouldn't the children inside the popover still get rendered even with null renderContent?

Reply:
@willm30  Seems like some of this was leftover from the `InfoPopover`, I simplified this and it should address your concerns now